From 71287b3610a5deb84da2686dc117b92867489284 Mon Sep 17 00:00:00 2001
From: arch1t3cht <arch1t3cht@gmail.com>
Date: Sun, 12 May 2024 16:17:43 +0200
Subject: [PATCH 1/2] Remove ccf and fm enums from vs_proxy

These values were renamed in VapourSynth R67.
Furthermore, they weren't supposed to be public API in the first place
(http://www.vapoursynth.com/2024/05/r67-r68-a-quick-succession-of-bug-fixes/)
and aren't used in any JET package except for being forwarded by
vs_proxy, so it's better to just remove them here.
---
 vstools/utils/vs_proxy.py | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/vstools/utils/vs_proxy.py b/vstools/utils/vs_proxy.py
index 9aeb0a8..11ccc5a 100644
--- a/vstools/utils/vs_proxy.py
+++ b/vstools/utils/vs_proxy.py
@@ -33,8 +33,8 @@
     EnvironmentPolicy, EnvironmentPolicyAPI, Error, FieldBased, FilterMode, FrameProps, FramePtr, Func, FuncData,
     Function, LogHandle, MatrixCoefficients, MediaType, MessageType, Plugin, RawFrame, RawNode, SampleType,
     TransferCharacteristics, VideoFormat, VideoFrame, VideoNode, VideoOutputTuple, __api_version__, __version__,
-    _CoreProxy, ccfDisableAutoLoading, ccfDisableLibraryUnloading, ccfEnableGraphInspection, clear_output,
-    clear_outputs, fmFrameState, fmParallel, fmParallelRequests, fmUnordered, get_current_environment, get_output,
+    _CoreProxy, clear_output,
+    clear_outputs, get_current_environment, get_output,
     get_outputs, has_policy, register_on_destroy, register_policy, unregister_on_destroy
 )
 
@@ -108,9 +108,9 @@
     'YUV444P13', 'YUV444P14', 'YUV444P15', 'YUV444P16', 'YUV444P17', 'YUV444P18', 'YUV444P19', 'YUV444P20', 'YUV444P21',
     'YUV444P22', 'YUV444P23', 'YUV444P24', 'YUV444P25', 'YUV444P26', 'YUV444P27', 'YUV444P28', 'YUV444P29', 'YUV444P30',
     'YUV444P31', 'YUV444P32', 'YUV444P8', 'YUV444P9', 'YUV444PH', 'YUV444PS', '_CoreProxy', '__all__',
-    '__api_version__', '__version__', 'ccfDisableAutoLoading', 'ccfDisableLibraryUnloading',
-    'ccfEnableGraphInspection', 'clear_output', 'clear_outputs', 'construct_parameter', 'construct_signature',
-    'construct_type', 'core', 'fmFrameState', 'fmParallel', 'fmParallelRequests', 'fmUnordered',
+    '__api_version__', '__version__',
+    'clear_output', 'clear_outputs', 'construct_parameter', 'construct_signature',
+    'construct_type', 'core',
     'get_current_environment', 'get_output', 'get_outputs', 'has_policy', 'pyx_capi', 'register_on_creation',
     'register_on_destroy', 'register_policy', 'try_enable_introspection', 'unregister_on_creation',
     'unregister_on_destroy', 'vs_file', 'clear_cache'

From c09c52541bd418840bfd443ff358bac51787fa8b Mon Sep 17 00:00:00 2001
From: arch1t3cht <arch1t3cht@gmail.com>
Date: Sun, 12 May 2024 16:18:32 +0200
Subject: [PATCH 2/2] Reformat after the previous commit

---
 vstools/utils/vs_proxy.py | 13 +++++--------
 1 file changed, 5 insertions(+), 8 deletions(-)

diff --git a/vstools/utils/vs_proxy.py b/vstools/utils/vs_proxy.py
index 11ccc5a..921af35 100644
--- a/vstools/utils/vs_proxy.py
+++ b/vstools/utils/vs_proxy.py
@@ -33,8 +33,7 @@
     EnvironmentPolicy, EnvironmentPolicyAPI, Error, FieldBased, FilterMode, FrameProps, FramePtr, Func, FuncData,
     Function, LogHandle, MatrixCoefficients, MediaType, MessageType, Plugin, RawFrame, RawNode, SampleType,
     TransferCharacteristics, VideoFormat, VideoFrame, VideoNode, VideoOutputTuple, __api_version__, __version__,
-    _CoreProxy, clear_output,
-    clear_outputs, get_current_environment, get_output,
+    _CoreProxy, clear_output, clear_outputs, get_current_environment, get_output,
     get_outputs, has_policy, register_on_destroy, register_policy, unregister_on_destroy
 )
 
@@ -108,12 +107,10 @@
     'YUV444P13', 'YUV444P14', 'YUV444P15', 'YUV444P16', 'YUV444P17', 'YUV444P18', 'YUV444P19', 'YUV444P20', 'YUV444P21',
     'YUV444P22', 'YUV444P23', 'YUV444P24', 'YUV444P25', 'YUV444P26', 'YUV444P27', 'YUV444P28', 'YUV444P29', 'YUV444P30',
     'YUV444P31', 'YUV444P32', 'YUV444P8', 'YUV444P9', 'YUV444PH', 'YUV444PS', '_CoreProxy', '__all__',
-    '__api_version__', '__version__',
-    'clear_output', 'clear_outputs', 'construct_parameter', 'construct_signature',
-    'construct_type', 'core',
-    'get_current_environment', 'get_output', 'get_outputs', 'has_policy', 'pyx_capi', 'register_on_creation',
-    'register_on_destroy', 'register_policy', 'try_enable_introspection', 'unregister_on_creation',
-    'unregister_on_destroy', 'vs_file', 'clear_cache'
+    '__api_version__', '__version__', 'clear_output', 'clear_outputs', 'construct_parameter', 'construct_signature',
+    'construct_type', 'core', 'get_current_environment', 'get_output', 'get_outputs', 'has_policy',
+    'pyx_capi', 'register_on_creation', 'register_on_destroy', 'register_policy', 'try_enable_introspection',
+    'unregister_on_creation', 'unregister_on_destroy', 'vs_file', 'clear_cache'
 ]
 
 
